charts/gardener-extension-shoot-lakom-service/templates/deployment.yaml

```diff
@@ -64,7 +64,6 @@ spec:
       serviceAccountName: gardener-extension-{{ .Values.serviceName }}
       securityContext:
         runAsNonRoot: true
-        allowPrivilegeEscalation: false
```

Comment:
This field seems like it is not part of the pod security context. How was it working before?